README.adoc

```diff
@@ -87,6 +87,13 @@ To deploy the server in your Kubernetes cluster, run:
 $ kubectl create -f https://raw.github.com/astefanutti/kubebox/master/kubernetes.yaml
 ```
 
+This will create the namespace `kubebox` and bring up all components in there
```

Comment:
Could you remove the comment about replacing the subject service account namespace?

Reply:
I have removed the comment and rephrased the note on how to stop the server